src/components/Components/ComponentDetails/tabs/useRegistryLoginUrl.ts

```diff
@@ -12,6 +12,6 @@ import { useKonfluxPublicInfo } from '~/hooks/useKonfluxPublicInfo';
 export const useRegistryLoginUrl = (): [string | null, boolean, unknown] => {
   const [publicInfo, loaded, error] = useKonfluxPublicInfo();
 
-  const url = publicInfo?.imageProxyUrl ? `${publicInfo.imageProxyUrl}/idp` : null;
+  const url = publicInfo?.imageProxyUrl ? `${publicInfo.imageProxyUrl}/oauth` : null;
```

Comment:
This `/oauth` is getting removed as part of `getRegistryUrlFromOAuthUrl`. No need to add here. This hook can just return the image proxy url